modules/security-context-constraints-psa-opting.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * authentication/managing-security-context-constraints.adoc
+
+:_content-type: PROCEDURE
+[id="security-context-constraints-psa-opting_{context}"]
+= Disabling pod security admission synchronization
+
+You can disable automatic pod security admission synchronization for a namespace.
```

Comment:
What about the namespaces that are disabled by default? And those that cannot ever be synchronized?

Reply:
It's a discrete module that describes the task of disabling the feature in namespaces where it is active. 

I can do one of two things for your first case:
* Change this to a more generic "setting synchronization" module, which could feel clunky in the end. 
* Add an "Enabling pod security admission synchronization" module, which would be quite similar to this one. 

The second case should be covered by `modules/security-context-constraints-psa-synchronization.adoc`, but we can nod to namespaces that cannot ever be synchronized here, too. 

Thoughts?

Reply:
Trying out the second approach. See `modules/security-context-constraints-psa-opting-in.adoc`. @stlaz

Reply:
As I'm likely not going to be owning this long-term, I'm game for that. 👍